internal/test-roborazzi/build.gradle.kts

```diff
@@ -1,25 +1,63 @@
+import coil3.addAllMultiplatformTargets
 import coil3.androidLibrary
+import coil3.androidUnitTest
+import org.jetbrains.kotlin.gradle.ExperimentalKotlinGradlePluginApi
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
 
 plugins {
     id("com.android.library")
-    id("kotlin-android")
+    id("kotlin-multiplatform")
     id("org.jetbrains.compose")
     id("org.jetbrains.kotlin.plugin.compose")
     id("io.github.takahirom.roborazzi")
 }
 
+addAllMultiplatformTargets()
 androidLibrary(name = "coil3.test.roborazzi")
 
-dependencies {
-    api(projects.coilCore)
+kotlin {
+    jvm {
+        @OptIn(ExperimentalKotlinGradlePluginApi::class)
+        compilerOptions {
+            freeCompilerArgs.add("-Xcontext-receivers")
+        }
+    }
 
-    implementation(projects.coilComposeCore)
-    implementation(projects.coilTest)
+    sourceSets {
+        val jvmCommonTest by getting
+        val jvmTest by getting
 
-    testImplementation(projects.internal.testUtils)
-    testImplementation(libs.bundles.test.jvm)
-    testImplementation(libs.roborazzi.compose)
-    testImplementation(libs.roborazzi.core)
-    testImplementation(libs.roborazzi.junit)
-    testImplementation(compose.desktop.uiTestJUnit4)
+        androidUnitTest.dependencies {
+            implementation(libs.roborazzi.compose)
+            implementation(libs.roborazzi.core)
+            implementation(libs.roborazzi.junit)
+        }
+        commonTest.dependencies {
+            api(projects.coilCore)
+            implementation(projects.coilComposeCore)
+            implementation(projects.coilTest)
+            implementation(projects.internal.testUtils)
+        }
+        jvmCommonTest.dependencies {
+            implementation(compose.desktop.uiTestJUnit4)
+            implementation(libs.bundles.test.jvm)
+        }
+        jvmTest.dependencies {
+            implementation(compose.desktop.currentOs)
+            implementation(libs.coroutines.swing)
+            implementation(libs.roborazzi.compose.desktop)
+        }
+    }
+}
+
+roborazzi {
+    // RoborazziOptions.RecordOptions.outputDirectoryPath and roborazzi.output.dir seems to be ignored on desktop.
+    // To workaround that this is used.
+    outputDir = layout.projectDirectory.dir("src/jvmTest/snapshots/images")
+}
+
+tasks.withType<KotlinCompile>().configureEach {
+    compilerOptions {
+        freeCompilerArgs.add("-Xcontext-receivers")
```

Comment:
Do we need to add `"-Xcontext-receivers"` here and above?

Reply:
Sorry for the inconvenience. I didn't think context receivers would take this long to be released. I'm willing to use normal parameters instead of context receivers, but for now, they are needed.

Reply:
No prob! Was mostly curious if we can declare it in only one spot.